lib/CandidateInfo.js

```diff
@@ -156,11 +156,16 @@
 
 /**
  * Expands a plain JSON object containing an CandidateInfo
- * @param {CandidateInfoPlain} plain JSON object
+ * @param {CandidateInfoPlain | CandidateInfo} plain JSON object
  * @returns {CandidateInfo} Parsed Candidate info
  */
 CandidateInfo.expand = function(plain)
 {
+	//Ensure it is not already a CandidateInfo object
+	if (plain.constructor.name === "CandidateInfo")
+		// @ts-ignore
+		return plain;
```

Comment:
here instead of muting the error, we can cast `plain` to the type we know it is:

```suggestion
		return /** @type {CandidateInfo} */ (plain);
```